Source/DafnyCore/Compilers/SinglePassCompiler.cs

```diff
@@ -1959,7 +1961,7 @@ protected class NullClassWriter : IClassWriter {
           } else if (m.Body == null && !(c is TraitDecl && !m.IsStatic) &&
                      !(!DafnyOptions.O.DisallowExterns && (Attributes.Contains(m.Attributes, "dllimport") || (IncludeExternMembers && Attributes.Contains(m.Attributes, "extern"))))) {
             // A (ghost or non-ghost) method must always have a body, except if it's an instance method in a trait.
-            if (Attributes.Contains(m.Attributes, "axiom") || (!DafnyOptions.O.DisallowExterns && Attributes.Contains(m.Attributes, "extern"))) {
+            if (Attributes.Contains(m.Attributes, "axiom")) {
```

Comment:
Why is this necessary?

Reply:
It is not - I must have been using it before. Changed!